jobserver/views/status.py

```diff
@@ -10,6 +10,19 @@
 from ..models import Backend
 
 
+class DBAvailability(View):
+    def get(self, request, *args, **kwargs):
+        backend = get_object_or_404(Backend, slug=self.kwargs["backend"])
+
+        if backend.slug != "tpp":
+            return HttpResponse()
+
+        if backend.jobrunner_state.get("mode", {}).get("v", "") == "db-maintenance":
+            return HttpResponse(status=503)
+
+        return HttpResponse()
```

Comment:
Can we return a plain text reponse here for human debuggers?

e.g.
```
"DB maintenance mode not implemented in {backend.name}"
"DB maintenance mode off (since {mode['ts']})"
"DB maintenance mode on (since {mode['ts']})"
```